etp-backend/src/main/clj/solita/etp/schema/geo.clj

```diff
@@ -9,3 +9,6 @@
    :maa                 schema/Str})
 
 (def Toimintaalue common-schema/Luokittelu)
+
+(def Country (assoc common-schema/Luokittelu
```

Comment:
Formatting seems a bit weird for me. Everything to single line all all args to assoc aligned vertically?

Reply:
Or if multiple lines then :id and schema/Str on sepatate lines?